collector/fixtures/e2e-64k-page-output.txt

```diff
@@ -3040,24 +3040,30 @@ node_softirqs_total{vector="rcu"} 508444
 node_softirqs_total{vector="sched"} 622196
 node_softirqs_total{vector="tasklet"} 1.783454e+06
 node_softirqs_total{vector="timer"} 1.481983e+06
+# HELP node_softnet_backlog_len_total Softnet backlog status
+# TYPE node_softnet_backlog_len_total counter
+node_softnet_backlog_len_total{cpu="0"} 0
+# HELP node_softnet_cpu_collision_total Number of collision occur while obtaining device lock while transmitting
+# TYPE node_softnet_cpu_collision_total counter
+node_softnet_cpu_collision_total{cpu="0"} 0
 # HELP node_softnet_dropped_total Number of dropped packets
 # TYPE node_softnet_dropped_total counter
 node_softnet_dropped_total{cpu="0"} 0
-node_softnet_dropped_total{cpu="1"} 41
-node_softnet_dropped_total{cpu="2"} 0
-node_softnet_dropped_total{cpu="3"} 0
+# HELP node_softnet_flow_limit_count_total Number of times flow limit has been reached
+# TYPE node_softnet_flow_limit_count_total counter
+node_softnet_flow_limit_count_total{cpu="0"} 0
 # HELP node_softnet_processed_total Number of processed packets
 # TYPE node_softnet_processed_total counter
 node_softnet_processed_total{cpu="0"} 299641
-node_softnet_processed_total{cpu="1"} 916354
-node_softnet_processed_total{cpu="2"} 5.577791e+06
-node_softnet_processed_total{cpu="3"} 3.113785e+06
+# HELP node_softnet_received_rps_total Number of times cpu woken up received_rps
+# TYPE node_softnet_received_rps_total counter
+node_softnet_received_rps_total{cpu="0"} 0
 # HELP node_softnet_times_squeezed_total Number of times processing packets ran out of quota
 # TYPE node_softnet_times_squeezed_total counter
 node_softnet_times_squeezed_total{cpu="0"} 1
-node_softnet_times_squeezed_total{cpu="1"} 10
-node_softnet_times_squeezed_total{cpu="2"} 85
-node_softnet_times_squeezed_total{cpu="3"} 50
+# HELP node_softnet_width softnet_data's Width
+# TYPE node_softnet_width gauge
+node_softnet_width{cpu="0"} 11
```

Comment:
What happened to the other CPUs? There should be four here.

Reply:
i just understand the bug while writing the comment, the CI failed if there was four CPU
https://app.circleci.com/pipelines/github/remijouannet/node_exporter/3/workflows/9e1efabb-3c19-4dca-ba9a-6682ac3889e6/jobs/11
https://app.circleci.com/pipelines/github/remijouannet/node_exporter/3/workflows/2bab6f97-2175-4852-8003-0a44ed06e1b5/jobs/7

i think the bug come from the switch to cpuStats.Index, every line probably have the cpuStats.index to 0

Reply:
i pushed a fix for cpuStats.Index, it is now only use if width is >= 13, e2e was 11 width so the index was always 0 for each CPU